context/app/static/js/components/organ/Samples/Samples.jsx

```diff
@@ -79,10 +77,32 @@ function SampleHeaderCell({ column, setSort, sortState }) {
   );
 }
 
+function SampleRow({ virtualRow, index, hits }) {
+  const {
+    _id: uuid,
+    _source: { hubmap_id, donor, descendant_counts, last_modified_timestamp },
+  } = hits[virtualRow.index];
+  return (
+    <TableRow sx={{ height: virtualRow.size }} data-testid={index}>
```

Comment:
We could make the test ID more descriptive, e.g. `sample-table-row-${index}` or something along those lines.

Reply:
Ah sorry this was a relic from debugging, thanks for noticing this!